docs/docs/Components/components-logic.md

```diff
@@ -0,0 +1,159 @@
+# Logic
+
+## Conditional router
+
+This component routes an input message to a corresponding output based on text comparison.
+
+The ConditionalRouterComponent routes messages based on text comparison. It evaluates a condition by comparing two text inputs using a specified operator and routes the message accordingly.
+
+### Parameters
+
+#### Inputs
+
+| Name           | Type     | Description                                                       |
+|----------------|----------|-------------------------------------------------------------------|
+| input_text     | String   | The primary text input for the operation.                         |
+| match_text     | String   | The text input to compare against.                                |
+| operator       | Dropdown | The operator to apply for comparing the texts.                    |
+| case_sensitive | Boolean  | If true, the comparison will be case sensitive.                   |
+| message        | Message  | The message to pass through either route.                         |
+| max_iterations | Integer  | The maximum number of iterations for the conditional router.      |
+| default_route  | Dropdown | The default route to take when max iterations are reached.        |
+
+#### Outputs
+
+| Name         | Type    | Description                                |
+|--------------|---------|--------------------------------------------|
+| true_result  | Message | The output when the condition is true.     |
+| false_result | Message | The output when the condition is false.    |
+
+## Data conditional router
+
+This component routes `Data` objects based on a condition applied to a specified key, including boolean validation.
+
+This component is particularly useful in workflows that require conditional routing of complex data structures, enabling dynamic decision-making based on data content.
+
+### Parameters
+
+#### Inputs
+
+| Name          | Type     | Description                                                                       |
+|---------------|----------|-----------------------------------------------------------------------------------|
+| data_input    | Data     | The Data object or list of Data objects to process.                               |
+| key_name      | String   | The name of the key in the Data object(s) to check.                               |
```

Comment:
Why is data capitalized in the description column? Also, it's not our standard to use "(s)" to make a word plural.